src/main/resources/assets/carpet-redcraft-addons/datapacks/RedPack - Craftable Sculk Sensor/pack.mcmeta

```diff
@@ -0,0 +1,11 @@
+{
+    "pack": {
+        "pack_format": 7,
+        "description": "RedPack by BisUmTo, Gnottero & DukeEdivad05 - Craftable Sculk Sensor v0.1.0"
+    },
+    "redpack": {
+        "id": "128357",,
+        "image": "https://img.youtube.com/vi/0GiWWcVG6x8/maxresdefault.jpg",
```

Comment:
Campo obbligatorio "video" mancante.
In caso non ci sia un video, inserire "KVIxcL8rNC8"